pkg/operator/starter.go

```diff
@@ -255,7 +255,8 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		configInformers.Config().V1().APIServers(),
 		kubeInformersForNamespaces,
 	).WithConfigUpgradableController().
-		WithLogLevelController()
+		WithLogLevelController().
+		WithoutPruneController()
```

Comment:
please change it to `WithPruneController` and remove `pruneController := prune.NewPruneController` from OAS-O

Reply:
compare with CAO